### DISCUSSION TITLE
How to keep primitive custom attribute

### DISCUSSION THREAD

**Discussion by andy259** [zeux/meshoptimizer#471] [Q&A]
Hi,
I hvae a custom attribute `_BATCHID` in the primitives. when I use ` -tc -noq` to do it, the  `_BATCHID`  is gone.  
How can I keeep the  attribute `_BATCHID` ?  
Thanks.
```
"primitives": [
    {
      "mode": 4,
      "attributes": {
        "POSITION": 376,
        "NORMAL": 377,
        "TEXCOORD_0": 378,
        "_BATCHID": 379
      },
      "material": 0
    }
  ]
```

**Comment by zeux** [comment]
Right now gltfpack doesn't preserve unknown attributes. Is `_BATCHID` related to 3D tiles or some extension like this? If this is common enough it might make sense to support out of the box, but I'd need a test model for this.
